### ISSUE TITLE
OSGI plugins are not working in the release branch 

### ISSUE THREAD

**Issue by bryanboza** [dotCMS/core#20329]
If you try to upload plugins in the release branch, those are not working. Maybe this is related to my environment configuration b/c is in Java 8 

Error: https://gist.github.com/bryanboza/5565325bd2d77bb9620230ead7df25291

**To Reproduce**
- Start the instance 
- Clone the plugins repo
- Deploy one of those 
- Try to upload from the plugins portlet 

**Expected behavior**
Every example should work without problems 

**Desktop (please complete the following information):**
Tested on release-21.05 // Tomcat9 // Java8

**Comment by dsilvam** [comment]
**Note to QA:**
Please test both `portlet` and `job` OSGI plugins activate and work as expected under the following scenarios. A checklist is included to help. 

**Running on Distro** (`./gradlew createDist`)

Test on both mac/unix and Windows.

1. **Running on the distro and Java 8** (
 
- [ ] Byte-buddy 1.6.12 should be used. Check for this message on the startup output _Using Byte-Buddy Version: 1.6.12_ . 
- [ ] Check the two byte-buddy jars are located under `{distroLocation}/dotserver/tomcat-9.0.41/webapps/ROOT/WEB-INF/lib`. 
Jars: 
* `{distroLocation}/dotserver/tomcat-9.0.41/webapps/ROOT/WEB-INF/lib/byte-buddy-1.6.12.jar`
* `{distroLocation}/dotserver/tomcat-9.0.41/webapps/ROOT/WEB-INF/lib/byte-buddy-agent-1.6.12.jar`
- [ ] [Portlet](https://github.com/dotCMS/plugin-seeds/tree/release-21.05/OSGi/com.dotcms.portlet) plugin activates and works as expected
- [ ] [Job](https://github.com/dotCMS/plugin-seeds/tree/release-21.05/OSGi/com.dotcms.job) plugin activates and works as expected 

2. **Running on the distro and Java 11**
 
- [ ] Byte-buddy 1.9.0 should be used. Check for this message on the startup output _Using Byte-Buddy Version: 1.9.0_ . 
- [ ] Check the two byte-buddy jars are located under `{distroLocation}/dotserver/tomcat-9.0.41/webapps/ROOT/WEB-INF/lib`. 
Jars: 
* `{distroLocation}/dotserver/tomcat-9.0.41/webapps/ROOT/WEB-INF/lib/byte-buddy-1.9.0.jar`
* `{distroLocation}/dotserver/tomcat-9.0.41/webapps/ROOT/WEB-INF/lib/byte-buddy-agent-1.9.0.jar`
- [ ] [Portlet](https://github.com/dotCMS/plugin-seeds/tree/release-21.05/OSGi/com.dotcms.portlet) plugin activates and works as expected
- [ ] [Job](https://github.com/dotCMS/plugin-seeds/tree/release-21.05/OSGi/com.dotcms.job) plugin activates and works as expected 

**Running from source** (`./gradlew deployWarTomcat`) 
Test on mac/unix

1. **Running from Source and Java 8**
 
- [ ] Byte-buddy 1.6.12 should be used. Check for this message on the startup output _Using Byte-Buddy Version: 1.6.12_ . 
- [ ] Check the two byte-buddy jars are located under `tomcat-9.0.41/webapps/ROOT/WEB-INF/lib`. 
Jars: 
* `{pathToTomcat}/webapps/ROOT/WEB-INF/lib/byte-buddy-1.6.12.jar`
* `{pathToTomcat}//webapps/ROOT/WEB-INF/lib/byte-buddy-agent-1.6.12.jar`
- [ ] [Portlet](https://github.com/dotCMS/plugin-seeds/tree/release-21.05/OSGi/com.dotcms.portlet) plugin activates and works as expected
- [ ] [Job](https://github.com/dotCMS/plugin-seeds/tree/release-21.05/OSGi/com.dotcms.job) plugin activates and works as expected 

2. **Running on the distro and Java 11**
 
- [ ] Byte-buddy 1.9.0 should be used. Check for this message on the startup output _Using Byte-Buddy Version: 1.9.0_ . 
- [ ] Check the two byte-buddy jars are located under `tomcat-9.0.41/webapps/ROOT/WEB-INF/lib`. 
Jars: 
* `{pathToTomcat}/webapps/ROOT/WEB-INF/lib/byte-buddy-1.9.0.jar`
* `{pathToTomcat}//webapps/ROOT/WEB-INF/lib/byte-buddy-agent-1.9.0.jar`
- [ ] [Portlet](https://github.com/dotCMS/plugin-seeds/tree/release-21.05/OSGi/com.dotcms.portlet) plugin activates and works as expected
- [ ] [Job](https://github.com/dotCMS/plugin-seeds/tree/release-21.05/OSGi/com.dotcms.job) plugin activates and works as expected

**Comment by victoralfaro-dotcms** [comment]
**2**
QA

**Java 8**
```
Using BYTE_BUDDY_VERSION: 1.6.12
```

**Portlet OSGI Plugin**
```
13:58:24.748  INFO  AJAX.OSGIAJAX - OSGI Bundle com.dotcms.portlet-0.2.jar Uploaded
Installing bundle com.dotcms.portlet / 0.2.0
13:58:28.415  INFO  servlet.ServletToolboxManager - Trying config file '/WEB-INF/toolbox.xml'
13:58:28.549  INFO  servlet.ServletToolboxManager - Toolbox setup complete.
13:58:28.710  INFO  util.GeoIp2CityDbUtil - Connecting to GeoIP2 database:/Users/victoralfaro/dev/dotcms/tomcat9/webapps/ROOT/WEB-INF/geoip2/GeoLite2-City.mmdb
13:58:28.768  WARN  rules.g - Actionlet with name 'CountRulesActionlet' already registered.
13:58:28.768  WARN  rules.g - Actionlet with name 'SetSessionAttributeActionlet' already registered.
13:58:28.768  WARN  rules.g - Actionlet with name 'SetRequestAttributeActionlet' already registered.
13:58:28.768  WARN  rules.g - Actionlet with name 'SetResponseHeaderActionlet' already registered.
13:58:28.768  WARN  rules.g - Actionlet with name 'PersonaActionlet' already registered.
13:58:28.768  WARN  rules.g - Actionlet with name 'VisitorTagsActionlet' already registered.
13:58:28.768  WARN  rules.g - Actionlet with name 'SendRedirectActionlet' already registered.
13:58:28.785  INFO  portlet.Activator - Injecting: com.dotmarketing.osgi.portlet.HelloWorldAction into classloader: ParallelWebappClassLoader
  context: ROOT
  delegate: true
----------> Parent Classloader:
java.net.URLClassLoader@685cb137

13:58:29.402  INFO  portlet.Activator - Added Struts Action Mapping: com.dotmarketing.osgi.portlet.HelloWorldAction
13:58:29.461  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
13:58:29.462  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
13:58:29.463  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
13:58:29.467  INFO  portlet.Activator - Added Portlet: EXT_STRUTS_HELLO_WORLD
13:58:29.473  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
13:58:29.513  INFO  portlet.Activator - Added Portlet: EXT_HELLO_WORLD
13:58:29.514  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
13:58:29.515  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
13:58:29.515  INFO  portlet.Activator - Added Portlet: EXT_JSP_HELLO_WORLD
13:58:29.516  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
13:58:29.579  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
Started bundle: file:/Users/victoralfaro/dev/dotcms/tomcat9/webapps/ROOT/WEB-INF/felix/load/com.dotcms.portlet-0.2.jar
13:58:32.642  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
13:58:32.647  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
```

**Job OSGI Plugin**
```
14:05:45.168  INFO  AJAX.OSGIAJAX - OSGI Bundle com.dotcms.job-0.2.jar Uploaded
14:05:46.372  INFO  cluster.ClusterFactory - Cluster topography has changed. This might force a rewire
14:05:46.372  INFO  cluster.ClusterFactory - Expecting Servers:[]
14:05:46.372  INFO  cluster.ClusterFactory - Found Servers:[]
14:05:46.372  INFO  cluster.ClusterFactory - Cluster Active. Rewiring
Installing bundle com.dotcms.job / 0.2.0
14:05:49.359  INFO  job.Activator - Injecting: com.dotmarketing.osgi.job.CustomJob into classloader: ParallelWebappClassLoader
  context: ROOT
  delegate: true
----------> Parent Classloader:
java.net.URLClassLoader@685cb137

14:05:49.483  INFO  job.Activator - Injecting: com.dotmarketing.osgi.job.TestClass into classloader: ParallelWebappClassLoader
  context: ROOT
  delegate: true
----------> Parent Classloader:
java.net.URLClassLoader@685cb137

14:05:49.506  INFO  job.Activator - Injecting: com.dotmarketing.osgi.job.CustomJob into classloader: ParallelWebappClassLoader
  context: ROOT
  delegate: true
----------> Parent Classloader:
java.net.URLClassLoader@685cb137

Started bundle: file:/Users/victoralfaro/dev/dotcms/tomcat9/webapps/ROOT/WEB-INF/felix/load/com.dotcms.job-0.2.jar
14:05:49.740  INFO  job.Activator - Added Quartz Job: Custom Job
14:05:50.052  INFO  job.CustomJob - ------------------------------------------
14:05:50.052  INFO  job.CustomJob - Start custom job
14:05:50.052  INFO  job.CustomJob -
14:05:50.052  INFO  job.CustomJob - param1: value1
14:05:50.052  INFO  job.CustomJob - param2: value2
14:05:50.052  INFO  job.CustomJob -
14:05:50.052  INFO  job.TestClass - Printing from TestClass.printA
14:05:50.052  INFO  job.TestClass - Printing from TestClass.printB
14:05:50.052  INFO  job.CustomJob -
14:05:50.052  INFO  job.CustomJob - Finish custom job (osgi version).
14:05:50.052  INFO  job.CustomJob - ------------------------------------------
```

**Java 11**
```
Using BYTE_BUDDY_VERSION: 1.9.0
```

**Portlet OSGI Plugin**
``` 14:29:19.396  INFO  AJAX.OSGIAJAX - OSGI Bundle com.dotcms.portlet-0.2.jar Uploaded
14:29:20.046  INFO  job.CustomJob - ------------------------------------------
14:29:20.046  INFO  job.CustomJob - Start custom job
14:29:20.046  INFO  job.CustomJob -
14:29:20.046  INFO  job.CustomJob - param1: value1
14:29:20.046  INFO  job.CustomJob - param2: value2
14:29:20.046  INFO  job.CustomJob -
14:29:20.046  INFO  job.TestClass - Printing from TestClass.printA
14:29:20.046  INFO  job.TestClass - Printing from TestClass.printB
14:29:20.046  INFO  job.CustomJob -
14:29:20.046  INFO  job.CustomJob - Finish custom job (osgi version).
14:29:20.046  INFO  job.CustomJob - ------------------------------------------
Installing bundle com.dotcms.portlet / 0.2.0
14:29:21.774  INFO  portlet.Activator - Injecting: com.dotmarketing.osgi.portlet.HelloWorldAction into classloader: ParallelWebappClassLoader
  context: ROOT
  delegate: true
----------> Parent Classloader:
java.net.URLClassLoader@76f2b07d

14:29:21.809  INFO  portlet.Activator - Added Struts Action Mapping: com.dotmarketing.osgi.portlet.HelloWorldAction
14:29:21.823  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
14:29:21.824  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
14:29:21.827  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
14:29:21.831  INFO  portlet.Activator - Added Portlet: EXT_STRUTS_HELLO_WORLD
14:29:21.838  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
14:29:21.982  INFO  portlet.Activator - Added Portlet: EXT_HELLO_WORLD
14:29:21.982  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
14:29:21.983  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
14:29:22.023  INFO  portlet.Activator - Added Portlet: EXT_JSP_HELLO_WORLD
14:29:22.024  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
14:29:22.024  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
Started bundle: file:/Users/victoralfaro/dev/dotcms/tomcat9/webapps/ROOT/WEB-INF/felix/load/com.dotcms.portlet-0.2.jar
14:29:24.998  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
14:29:25.128  INFO  caffine.CaffineCache - ***	 Building Cache : portletcache, size:1000,Concurrency:32
```

**Job OSGI Plugin**
```
14:26:25.690  INFO  AJAX.OSGIAJAX - OSGI Bundle com.dotcms.job-0.2.jar Uploaded
Installing bundle com.dotcms.job / 0.2.0
14:26:29.486  INFO  servlet.ServletToolboxManager - Trying config file '/WEB-INF/toolbox.xml'
14:26:29.577  INFO  servlet.ServletToolboxManager - Toolbox setup complete.
14:26:29.606  INFO  util.GeoIp2CityDbUtil - Connecting to GeoIP2 database:/Users/victoralfaro/dev/dotcms/tomcat9/webapps/ROOT/WEB-INF/geoip2/GeoLite2-City.mmdb
14:26:29.674  WARN  rules.g - Actionlet with name 'CountRulesActionlet' already registered.
14:26:29.674  WARN  rules.g - Actionlet with name 'SetSessionAttributeActionlet' already registered.
14:26:29.674  WARN  rules.g - Actionlet with name 'SetRequestAttributeActionlet' already registered.
14:26:29.674  WARN  rules.g - Actionlet with name 'SetResponseHeaderActionlet' already registered.
14:26:29.674  WARN  rules.g - Actionlet with name 'PersonaActionlet' already registered.
14:26:29.674  WARN  rules.g - Actionlet with name 'VisitorTagsActionlet' already registered.
14:26:29.674  WARN  rules.g - Actionlet with name 'SendRedirectActionlet' already registered.
14:26:29.675  INFO  job.Activator - Injecting: com.dotmarketing.osgi.job.CustomJob into classloader: ParallelWebappClassLoader
  context: ROOT
  delegate: true
----------> Parent Classloader:
java.net.URLClassLoader@76f2b07d

14:26:30.225  INFO  job.Activator - Injecting: com.dotmarketing.osgi.job.TestClass into classloader: ParallelWebappClassLoader
  context: ROOT
  delegate: true
----------> Parent Classloader:
java.net.URLClassLoader@76f2b07d

14:26:30.243  INFO  job.Activator - Injecting: com.dotmarketing.osgi.job.CustomJob into classloader: ParallelWebappClassLoader
  context: ROOT
  delegate: true
----------> Parent Classloader:
java.net.URLClassLoader@76f2b07d

Started bundle: file:/Users/victoralfaro/dev/dotcms/tomcat9/webapps/ROOT/WEB-INF/felix/load/com.dotcms.job-0.2.jar
14:26:30.403  INFO  job.Activator - Added Quartz Job: Custom Job
14:26:30.412  INFO  job.CustomJob - ------------------------------------------
14:26:30.412  INFO  job.CustomJob - Start custom job
14:26:30.412  INFO  job.CustomJob -
14:26:30.413  INFO  job.CustomJob - param1: value1
14:26:30.413  INFO  job.CustomJob - param2: value2
14:26:30.413  INFO  job.CustomJob -
14:26:30.413  INFO  job.TestClass - Printing from TestClass.printA
14:26:30.413  INFO  job.TestClass - Printing from TestClass.printB
14:26:30.413  INFO  job.CustomJob -
14:26:30.413  INFO  job.CustomJob - Finish custom job (osgi version).
14:26:30.413  INFO  job.CustomJob - ------------------------------------------
14:26:40.004  INFO  job.CustomJob - ------------------------------------------
14:26:40.004  INFO  job.CustomJob - Start custom job
14:26:40.004  INFO  job.CustomJob -
14:26:40.004  INFO  job.CustomJob - param1: value1
14:26:40.004  INFO  job.CustomJob - param2: value2
14:26:40.004  INFO  job.CustomJob -
14:26:40.004  INFO  job.TestClass - Printing from TestClass.printA
14:26:40.004  INFO  job.TestClass - Printing from TestClass.printB
14:26:40.004  INFO  job.CustomJob -
14:26:40.004  INFO  job.CustomJob - Finish custom job (osgi version).
14:26:40.004  INFO  job.CustomJob - ------------------------------------------ ```

**Comment by bryanboza** [comment]
Fixed, tested on release-21.05 // Postgres // FF. Tested using Java8 with Byte-Buddy Version: 1.6.12
